.github/workflows/push-asset-readme-update.yml

```diff
@@ -0,0 +1,16 @@
+name: Plugin asset/readme update
+on:
+  push:
+    branches:
+    - master
+jobs:
+  master:
+    name: Push to master
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: WordPress.org plugin asset/readme update
+      uses: 10up/actions-wordpress/dotorg-plugin-asset-update@master
```

Comment:
@helen did you want to use the new repo instead?

Reply:
🤦‍♀ yes